tests/_files/source_with_multiline_constant_return.php

```diff
@@ -244,4 +244,17 @@ public function Spaceship(): int
             1
         ;
     }
+
+    public function VariableInArray(): int
+    {
+        $n = 1;
+
+        return
+            [
+                2 + $n,
+                $n ,
+                2 . $n,
+            ]
+            ;
+    }
```

Comment:
```suggestion
    public function VariableInArray(): array
    {
        $a = 1;
        $b = 2;

        return
            [
                $a
                +
                $b
                +
                $b,
                $a ,
                2
                +
                $a
                +
                $b
                +
                $b,
                $a ,
                $a
                .
                $b
                .
                $b,
                2
                .
                $a
                .
                $b
                .
                $b,
            ]
            ;
    }
```

test with this example and expected output:
```
250,
251,
// line 255 must not be present as not present in xdebug output
257,
259,
260,
263,
265,
267,
268,
// line 269 must not be present as not present in xdebug output
271,
273,
276,
278,
280,
```

@derickr why lines 255 and 269 are not present in xdebug coverage output or is it a bug?

tested on PHP 8.1.12 Win x64 and xdebug 3.1.4 and xdebug 3.2.0RC1